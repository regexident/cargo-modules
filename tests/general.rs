#[macro_use]
mod util;

mod help {
    test_cmd!(
        args: "--help",
        success: true,
        color_mode: ColorMode::Plain,
        project: smoke
    );
}
