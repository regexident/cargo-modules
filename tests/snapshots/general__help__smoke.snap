---
source: tests/general.rs
expression: output
snapshot_kind: text
---
COMMAND:
--help

STDERR:

STDOUT:
Visualize/analyze a crate's internal structure.

Usage: cargo-modules <COMMAND>

Commands:
  structure     Prints a crate's hierarchical structure as a tree.
  dependencies  Prints a crate's internal dependencies as a graph.
  orphans       Detects unlinked source files within a crate's directory.
  help          Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version